src/main/target/BEEROTORF4/target.h

```diff
@@ -58,6 +58,7 @@
 #define MAX7456_DMA_IRQ_HANDLER_ID          DMA1_ST0_HANDLER
 
 #define USE_SDCARD
+#define USB_MSC
```

Comment:
I thought you wanted to convert to `USE_` before doing more changes concerning it.

Reply:
I wanted to, and I will. I just added this to not have to manually add it every time I am testing. ;-)